content/en/docs/armory-agent/_index.md

```diff
@@ -19,85 +19,86 @@ description: >
 
 * Security
   * Keep your Kubernetes API servers private from Spinnaker.
-  * Only the information Spinnaker needs leaves the cluster.
+  * Only the information Armory Enterprise needs leaves the cluster.
   * Decentralize your account management. Using Kubernetes Service Accounts, teams control what Spinnaker can do. Add or remove accounts in real time. and then use them without restarting Spinnaker.
   * Use Kubernetes Service Accounts or store your `kubeconfig` files in one of the supported [secret engines]({{< ref "secrets#supported-secret-engines" >}}), or provision them via the method of your choice as Kubernetes secrets.
 
 
 * Usability
-  * Use the Agent alongside Spinnaker and benefit from performance improvements.
+  * Use the Agent alongside Armory Enterprise and benefit from performance improvements.
   * Use the Agent in a target cluster and get Kubernetes accounts automatically registered.
   * Use YAML, a HELM chart, or a Kustomize template to inject the Agent into newly provisioned Kubernetes clusters, and immediately make those clusters software deployment targets.
-  * Use the Agent with little operational overhead or changes in the way you currently manage Spinnaker.
+  * Use the Agent with little operational overhead or changes in the way you currently manage Armory Enterprise.
 
-Check out the Quick Start [guide]({{< ref "armory-agent-quick" >}}) to deploy the Agent on your Kubernetes infrastructure.
+Check out the installation [guide]({{< ref "armory-agent-quick" >}}) for how to deploy the Agent components in Armory Enterprise and in your Kubernetes infrastructure.
 
 ## Deployment topologies
```

Comment:
Is it just me or is this a really odd/non-descriptive title for this section? I'm thrown off by the word topology.

Maybe something like `Deployment modes (or options)`?